core/DataAccess/LogAggregator.php

```diff
@@ -416,6 +416,7 @@ protected function getVisitsMetricFields()
             Metrics::INDEX_BOUNCE_COUNT                   => "sum(case " . self::LOG_VISIT_TABLE . ".visit_total_actions when 1 then 1 when 0 then 1 else 0 end)",
             Metrics::INDEX_NB_VISITS_CONVERTED            => "sum(case " . self::LOG_VISIT_TABLE . ".visit_goal_converted when 1 then 1 else 0 end)",
             Metrics::INDEX_NB_USERS                       => "count(distinct " . self::LOG_VISIT_TABLE . ".user_id)",
+            Metrics::INDEX_NB_PROFILABLE                  => "sum(" . self::LOG_VISIT_TABLE . ".profilable)",
```

Comment:
Shouldn't this use a `case` construct like the other sums above?

```suggestion
            Metrics::INDEX_NB_PROFILABLE                  => "sum(case " . self::LOG_VISIT_TABLE . ".profilable when 1 then 1 else 0 end)",
```

Reply:
It could, but it doesn't have to. profilable will only be 1 or 0. Like visit_goal_converted I guess, so I suppose that doesn't have to either. I can change it to be consistent.

Reply:
changed